src/auth/README.md

```diff
@@ -72,69 +83,196 @@ if (blockstack.isUserSignedIn()) {
 }
 ```
 
-Make sure your manifest.json file has appropriate CORS headers so that it can
+Make sure your `manifest.json` file has appropriate CORS headers so that it can
 be fetched via an http `GET` from any origin.
 
 
 7) Serve your application
 
-## Operations
+## User flow
 
-### Request a user to sign in
+What follows is a walk through of the experience of a user, Alice, signing in to your app with Blockstack. 
 
-```es6
-import { requestSignIn } from 'blockstack'
+First, Alice clicks the "Sign in with Blockstack" button on your app. She is redirected to her copy of the Blockstack Browser. The Blockstack Browser shows Alice an approval dialog with information about your app including:
```

Comment:
This section could be nice as a list:

1. Alice clicks the "Sign in with Blockstack" button on your app, AwesomeApp
2. Alice is redirected to her copy of the Blockstack Browser, showing Alice an approval dialog with information about AwesomeApp including:
    * The origin your app was served from
    * Your app's name
    * Your app's logo
    * The types of permissions and data your app is requesting

3. Alice chooses to authenticate with one of her Blockstack IDs by selecting the ID and clicking Approve.
4. Alice is redirected back to AwesomeApp
5. AwesomeApp receives:
    * a cryptographic proof of Alice's identity
    * public information in Alice's profile
    * access to a dedicated bucket in Alice's Gaia storage hub for application reads and writes